client/src/views/mobile/CellMobileView.tsx

```diff
@@ -38,97 +40,140 @@ const CellMobileView = ({
   confirmReport,
   confirmSacrifice,
   confirmReset,
+  bruteList
 }: CellMobileViewProps) => {
   const { t } = useTranslation();
   const { brute, owner } = useBrute();
+  const navigate = useNavigate();
   const theme = useTheme();
   const isXs = useMediaQuery(theme.breakpoints.only('xs'));
+  const pageRef = useRef<HTMLInputElement>(null);
+
+  useEffect(
+    () => {
+      let touchstartX = 0;
+      let touchendX = 0;
+      const checkSwipe = () => {
+        if (touchendX > touchstartX && touchendX - touchstartX > -120) {
+          if (bruteList && bruteList[bruteList.indexOf(brute!.name) - 1]) {
+            navigate(`/${bruteList[bruteList.indexOf(brute!.name) - 1]}/cell`);
+          }
+        }
+        if (touchendX < touchstartX && touchstartX - touchendX > -120) {
+          if (bruteList && bruteList[bruteList.indexOf(brute!.name) + 1]) {
+            navigate(`/${bruteList[bruteList.indexOf(brute!.name) + 1]}/cell`);
+          }
+        }
+      };
+
+      document.addEventListener('touchstart', (e) => {
+        touchstartX = e.changedTouches[0].screenX;
+      }, false);
+      document.addEventListener('touchmove', (e) => {
+        touchendX = e.changedTouches[0].screenX;
+        checkSwipe();
+      }, false);
+
+      return () => {
+        document.removeEventListener('touchstart', (e) => {
+          touchstartX = e.changedTouches[0].screenX;
+        });
+        document.removeEventListener('touchmove', (e) => {
+          touchendX = e.changedTouches[0].screenX;
+          checkSwipe();
+        });
+      };
+    },
+
+    [brute, bruteList, navigate]
+  );
 
   return brute && (
-    <Page title={`${brute.name || ''} ${t('MyBrute')}`} headerUrl={`/${brute.name}/cell`}>
-      <Grid container spacing={1} alignItems="center">
-        {/* BRUTE NAME + SOCIALS */}
-        <Grid item xs={12} sm={6} order={isXs ? 1 : 0}>
-          <CellSocials smallScreen />
-        </Grid>
-        <Grid item xs={12} sm={6} order={isXs ? 6 : 0} alignSelf="stretch">
-          {/* REF LINK + CLAN */}
-          <Paper sx={{
-            bgcolor: 'background.paperLight',
-            px: 0,
-            py: 1,
-            height: 'calc(100% - 32px)',
-            display: 'flex',
-            justifyContent: 'center',
-            alignContent: 'center',
-            flexDirection: 'column',
-            textAlign: 'center',
-          }}
-          >
-            <Tooltip title={t('refLink')}>
-              <Text bold center>{`${window.location.origin}?ref=${brute.name}`}</Text>
-            </Tooltip>
-            {(ownsBrute || !!brute.clanId) && (
+    <div>
```

Comment:
Why add a div?

You should try to alter the component structure as little as possible